README.md

```diff
@@ -2,12 +2,10 @@
 # Fixing Polygons in OSM
 
 There are many different ways (multi)polygons in OpenStreetMap can be mapped
-correctly and there are even more ways in which they can be mapped incorrectly.
-Of the over 220 million (multi)polygons in OSM more than 100,000 contain
-mapping errors of one kind or another and about 250,000 are tagged *old style*
-with tags on the outer ways instead of on the relation making multipolygon
-tagging and processing much more complicated and much more expensive than it
-needs to be.
+correctly and there are even more ways in which they can be mapped incorrectly, 
+using *old style* - with tags on the outer ways instead of on the relation 
+making multipolygon tagging and processing much more complicated and much more 
+expensive than it needs to be.
```

Comment:
@pnorman, @joto - I have to explain what I meant by "very misleading"

"[Of the over 220 million (multi)polygons in OSM](https://github.com/osmlab/fixing-polygons-in-osm#fixing-polygons-in-osm)" - but if you check a database [will get over 4M relations](https://taginfo.openstreetmap.org/relations).

So either: 1. you clarify how do you calculate "relation" or rather use real metric "number of ways without area=no tags + valid MP" or 2. remove unannotated (improperly described) numbers/stats

Thats why I'm suggesting to include links to database(s) and [stats](http://area.jochentopf.com/stats/).